src/subcategories/index.ts

```diff
@@ -0,0 +1,15 @@
+import { makeEnum } from '@transcend-io/type-utils';
+
+import { FinancialSubCategory } from './financial';
+import { CommunicationIdentifierType } from '../identifier';
+
+export const DefaultDataSubCategoryType = makeEnum({
+  ...CommunicationIdentifierType,
```

Comment:
I would create a new enum just for communication/contact subcategories. It seems like `CommunicationIdentifierType` is used by `IdentifierType` which we don't want to extend if we just wanted to add a new contact subcategory.

Reply:
I'm okay with that. It does mean that there will be some redundancy, but maybe that's okay if we're considering identifiers and datasubcategories to be essentially different and only coincidentally having some overlapping values.